modules/local/sanger_tol_btk.nf

```diff
@@ -0,0 +1,100 @@
+process SANGER_TOL_BTK {
+    tag "$meta.id"
+    label 'process_low'
+
+    input:
+    tuple val(meta), path(reference, stageAs: "REFERENCE.fa")
+    tuple val(meta1), path(bam) // Name needs to remain the same as previous process as they are referenced in the samplesheet
+    tuple val(meta2), path(samplesheet_csv, stageAs: "SAMPLESHEET.csv")
+    path blastp, stageAs: "blastp.dmnd"
+    path blastn
+    path blastx
+    path btk_config_file
+    path tax_dump
+    path btk_yaml, stageAs: "BTK.yaml"
+    val taxon
+    val gca_accession
+
+    output:
+    path("${meta.id}_btk_out/blobtoolkit/plots"),                   emit: btk_plots
+    path("${meta.id}_btk_out/blobtoolkit/draft"),                   emit: btk_dataset
+    path("${meta.id}_btk_out/blobtoolkit/draft/summary.json.gz"),   emit: btk_summary
+    path("${meta.id}_btk_out/busco"),                               emit: btk_busco
```

Comment:
This only the BUSCO run data without anything else. This should perhaps be called something other than `btk_busco` in order to not confuse it with the entire `sanger-tol/blobtoolkit` pipeline that is referred to as `btk_busco` in other places in ASCC code

Reply:
Renamed to:
```
    output:
    path("${meta.id}_btk_out/blobtoolkit/plots"),                   emit: plots
    path("${meta.id}_btk_out/blobtoolkit/draft"),                   emit: dataset
    path("${meta.id}_btk_out/blobtoolkit/draft/summary.json.gz"),   emit: summary_json
    path("${meta.id}_btk_out/busco"),                               emit: busco_data
    path("${meta.id}_btk_out/multiqc"),                             emit: multiqc_report
    path("blobtoolkit_pipeline_info"),                              emit: pipeline_info 
```

These are inherently the output of the module and should be better understood with the removal of `btk_`